functions_and_scope_lab.js

```diff
@@ -0,0 +1,363 @@
+// # Function Exercises
+//
+// ## Question 1
+//
+// Write a function that returns the largest of two numbers. If they are equal, return the second one.
+//
+// Example:
+//
+// Input: `12, 4`
+//
+// Output: `12`
+
+console.log("1._________________");
+
+function larger (num1, num2) {
```

Comment:
Since the question asked for returning a value, return would be more appropriate here rather than logging stuff.